.eslintignore

```diff
@@ -377,8 +377,7 @@ src/Explorer/Notebook/temp/inputs/editor.tsx
 src/Explorer/Notebook/temp/markdown-cell.tsx
 src/Explorer/Notebook/temp/source.tsx
 src/Explorer/Notebook/temp/syntax-highlighter/index.tsx
-src/Explorer/SplashScreen/SplashScreenComponent.tsx
-src/Explorer/SplashScreen/SplashScreenComponentApdapter.tsx
+src/Explorer/SplashScreen/SplashScreen.tsx
```

Comment:
Do you think its possible to remove this from eslintignore in this step or should it be out of scope for these migration PRs?

Reply:
Both! In the case of this file it was just a bunch of "don't pass null" warnings, so super easy. But it also muddies the waters, I think?

Reply:
Fair. I am fine with doing it later.